config/initializers/devise.rb

```diff
@@ -11,10 +11,17 @@
 
   config.mailer_sender = "support@unboxed.co"
 
-  config.mailer = "DeviseMailer"
+  require "application_mailer"
```

Comment:
Can this be done in a `to_prepare` block instead of with the explicit requires?

Reply:
Would you be able to pull this branch locally and give this a go? Think I tried but maybe was missing something